app/components/gh-members-lab-setting.js

```diff
@@ -93,11 +96,11 @@ export default Component.extend({
 
         return {
             monthly: {
-                amount: parseInt(monthly.amount) / 100 || 0,
+                amount: parseInt(monthly.amount) / 100 || 1,
                 currency: monthly.currency
             },
             yearly: {
-                amount: parseInt(yearly.amount) / 100 || 0,
+                amount: parseInt(yearly.amount) / 100 || 1,
```

Comment:
Should default values for monthly/yearly match [default settings](https://github.com/TryGhost/Ghost/blob/master/core/server/data/schema/default-settings.json#L249) of 5/50?